tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -1,6 +1,7 @@
 #define MAX_DATA_SIZE 1000
 
-/* Implementation of safe malloc which returns NULL if the reuested size is 0.*/
+/* Implementation of safe malloc which returns NULL if the requested size is 0.
+ Depending on the platform the malloc may return either NULL or an address. */
```

Comment:
Please be more specific: "Warning: The behavior of malloc(0) is platform dependent.  It is possible for malloc(0) to return an address without allocating memory."